src/main/resources/config.sample.yml

```diff
@@ -45,30 +48,45 @@ platforms :
       url: http://example.com/
 
     - type: pybossa
+
       # the key to access the pybossa api
       apiKey: 8ec92fa1-1bd1-42ad-8524-3d2bab4588b1
+
       # the url to the pybossa api
       url: http://localhost:5000/api
+
+      # name which is used to display
       name: Pybossa
+
       # the id of the pybossa project to use for all experiments
       projectId: 1
+
       # if calibrations are allowed on this platform
       calibrationsAllowed: true
 
 # basic configurations for the mail-module
-mail :
+mail:
```

Comment:
really, its not about this line, its about all changes to the file, do you really need to force everybodys on your personal prefered style? Can you please stop reformatting like that.